packages/knip/fixtures/plugins/nano-staged/.nano-staged.js

```diff
@@ -0,0 +1,8 @@
+export default{
+    '*.(js|ts|tsx)': async files => {
+        return ['eslint --max-warnings=0 --cache --fix .', 'prettier --write .'];
+      },
+      '*.!(js|ts|tsx)': files => {
+        return files.map(filename => `prettier --write --ignore-unknown '${filename}'`);
+      },
+}
```

Comment:
Please format new files according to project settings (either manually or temporarily remove `**/fixtures` from root `biome.json`)